apps/client/src/features/rundown/event-block/EventBlockInner.tsx

```diff
@@ -157,7 +157,7 @@ const EventBlockInner = (props: EventBlockInnerProps) => {
               }`}
             />
           </Tooltip>
-          <Tooltip label='Start time is later than end'>
+          <Tooltip label={timeStart > timeEnd ? 'Start time is later than end' : ''}>
             <div className={`${style.indicator} ${timeStart > timeEnd ? style.nextDay : ''}`} />
           </Tooltip>
```

Comment:
```suggestion
{timeStart > timeEnd && (
          <Tooltip label={'Start time is later than end'}>
            <div className={`${style.indicator} ${style.nextDay}`} />
          </Tooltip>
 )}
```

Question: I am not sure I understand the change.

If the element is visible, it should have a tooltip
If the element is not visible, is there a reason to render it? could we just conditionally render the whole thing?

Reply:
hmm they would then end up moving about depending on witch one is active
we could maybe place them with a css grid? or is it ok that they move?
![image](https://github.com/cpvalente/ontime/assets/19875125/0e65d590-5546-4917-8148-524fe7551810)